common/utils/src/main/resources/error/error-states.json

```diff
@@ -2682,7 +2682,7 @@
         "usedBy": ["SQL/Foundation"]
     },
     "33000": {
-        "description": "invalid SQL descriptor nameno subclass)",
+        "description": "invalid SQL descriptor nameno subclass",
```

Comment:
Is this change related to the current pr?

Reply:
It's not relevant, it's a small typo, just fixed.

Reply:
Sounds good, let me have a full check and separate a new PR.